### PR TITLE
Update candidate JSON files from Google sheet.

### PR DESCRIPTION
**Pull request by BolunThompson**
Solves issue #36. This is a script that creates and updates the JSON files for the candidates based on [this](https://docs.google.com/spreadsheets/d/1mENueYg0PhXE_MA9AypWWBJvBLdY03b8H_N_aIW-Ohw/edit#gid=0) Google sheet.
It normalizes the names by replacing spaces with underscores and converting the name to lower case. It also generates folders for the candidates based on their office.

### CODE REVIEW

File: src/assets/candidates/2020/mayor/barbara_bry/barbara_bry.json [hunk 1]
```diff
@@ -1,30 +1,53 @@
...
+      "in": "$0",
+      "out": "$0"
+    }
+  ],
+  "oppose": "$0",
+  "support": "$0",
+  "committee name": "Barbara Bry for San Diego Mayor 2020",
+  "first": "Barbara",
+  "last": "Bry"
+}
```
Comment (nancyheiss) [inline src/assets/candidates/2020/mayor/barbara_bry/barbara_bry.json:53]: Looks like we need a newline here.

File: src/scripts/calculation_download_gdrive_info.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import json
+import math
+import os
+import typing
+from urllib import request
+
+import pandas as pd
+
+
+def replace_nan(value, replace):
```
Comment (nancyheiss) [inline src/scripts/calculation_download_gdrive_info.py:11]: Are there any guidelines about how functions should be sorted in a file? I was sort of expecting these to be in alpha order but they aren't so I'm not sure what the best practice is. Also, seeing a class defined in the middle of the file surprised me as well but I'm really not all that familiar with how the contents of a module should be organized.

File: src/scripts/calculation_download_gdrive_info.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    try:
+        if math.isnan(value):
+            return replace
+    except TypeError:
+        pass
+    return value
+
+
+def read_candidate_csv(file):
```
Comment (nancyheiss) [inline src/scripts/calculation_download_gdrive_info.py:20]: The function names and variable names a good (descriptive) but I'm wondering whether or not we should have some docstrings in this file.

File: src/assets/candidates/2020/mayor/barbara_bry/barbara_bry.json (hunk 1 above)
Comment (BolunThompson) [reply src/assets/candidates/2020/mayor/barbara_bry/barbara_bry.json:53]: Will change, thanks!

File: src/scripts/calculation_download_gdrive_info.py (hunk 3 above)
Comment (BolunThompson) [reply src/scripts/calculation_download_gdrive_info.py:20]: That would be a good idea. I'll add some.

File: src/scripts/calculation_download_gdrive_info.py (hunk 2 above)
Comment (BolunThompson) [reply src/scripts/calculation_download_gdrive_info.py:11]: As far as I know, there are no guidelines about that. Personally, I don't stick to anything strictly, though I would be happy to start doing so. I don't know what is best practice specifically however. @tommy-stone is there any particular convention for this?

**Pull request merged**